benches/iai_stark.rs

```diff
@@ -0,0 +1,17 @@
+mod functions;
+mod util;
+
+#[inline(never)]
+fn simple_fibonacci_benchmarks() {
+    functions::stark::prove_fib(8);
```

Comment:
We should use `iai_callgrind::black_box` on these benchmarks
```suggestion
    black_box(functions::stark::prove_fib(black_box(8)));
```